geoserver/webapp/src/docker/Dockerfile

```diff
@@ -1,8 +1,17 @@
-FROM georchestra/base
+FROM jetty:9.3-jre8
+
+
+RUN apt-get update && \
+    apt-get install -y unzip && \
+    rm -rf /var/lib/apt/lists/*
 
 ADD . /
 ADD https://github.com/georchestra/geoserver_minimal_datadir/archive/master.zip /tmp/datadir.zip
```

Comment:
We should probably create a 15.12 branch in this repo, and target this branch here.